src/main/webapp/assets/feedStyle.css

```diff
@@ -0,0 +1,87 @@
+#page-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 80%;
+  margin-bottom: 100px;
+  margin-left: 10%;
+  margin-right: 10%;
+}
+
+/* Create Posts */
+#create-post-button {
+  background-color: #303030;
+  border: none;
+  border-radius: 50px;
+  box-shadow: 0 1px 10px lightgrey, 0 -1px 10px lightgrey;
+  color: white;
+  font-size: 24px;
+  font-weight: 400;
+  height: 60px;
+  margin-left: auto;
+  margin-right: 0;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  min-width: 100px;
```

Comment:
I recommend using rem over px for accessibility. Here's a great article that breaks it down: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px

Reply:
Great find! Just implemented the change